### PR TITLE
Add table busket and favorite

### PR DESCRIPTION
**Pull request by aliaksharbachou**

### CODE REVIEW

File: src/main/java/internship/mbicycle/storify/model/Basket.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    private long idUser;
+
+    private List<Product> productList;
+
+    public void addProductToProductList(Product product) {
+        if (productList == null) {
+            productList = new ArrayList<>();
+        }
+        productList.add(product);
+    }
```
Comment (OlgaEvdokimova) [inline src/main/java/internship/mbicycle/storify/model/Basket.java:27]: I assume I will be an Entity, so delete logic from Entity, all logic will be in Services

File: src/main/java/internship/mbicycle/storify/model/Favorite.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class Favorite {
+//    @Id
```
Comment (OlgaEvdokimova) [inline src/main/java/internship/mbicycle/storify/model/Favorite.java:15]: delete redundant commented code from Git, you can use local history

File: src/main/java/internship/mbicycle/storify/model/Basket.java (hunk 1 above)
Comment (aliaksharbachou) [reply src/main/java/internship/mbicycle/storify/model/Basket.java:27]: Done

File: src/main/java/internship/mbicycle/storify/model/Favorite.java (hunk 2 above)
Comment (aliaksharbachou) [reply src/main/java/internship/mbicycle/storify/model/Favorite.java:15]: Done

**Review by OlgaEvdokimova — APPROVED** [review]

**Pull request merged**